rules_building_block/defense_evasion_masquerading_windows_dll.toml

```diff
@@ -40,9 +40,24 @@ library where event.action == "load" and dll.Ext.relative_file_creation_time <=
         "?:\\$WINDOWS.~BT\\Work\\*",
         "?:\\Windows\\WinSxS\\*",
         "?:\\Windows\\SoftwareDistribution\\Download\\*",
-        "?:\\Windows\\assembly\\NativeImages_v*"
+        "?:\\Windows\\assembly\\NativeImages_v*", 
+        "E:\\docker\\windowsfilter\\*", 
+        "E:\\System\\Users\\*",
+        "\\\\?\\Volume{*}\\Windows\\*"
       )
     ) and
+    not process.executable : 
+              ("C:\\Packages\\Plugins\\Microsoft.GuestConfiguration.ConfigurationforWindows\\*\\dsc\\GC\\gc_*.exe", 
+               "?:\\Program Files\\*\\*.exe", 
+               "?:\\Program Files (x86)\\*\\*.exe", 
```

Comment:
```suggestion
```
I think we should avoid those big exceptions, as this is a BBR, we can afford some FPs

Reply:
avoiding those big exception generate lot of FPs, its not some FPs. this risk to miss true TPs due to high volume, also programfiles is writable by local admin (which limit the abuse scope).